app/src/main/java/net/gsantner/opoc/preference/SharedPreferencesPropertyBackend.java

```diff
@@ -202,11 +202,15 @@ public String getString(@StringRes int keyResourceId, @StringRes int defaultValu
     }
 
     public String getString(String key, String defaultValue, final SharedPreferences... pref) {
-        return gp(pref).getString(key, defaultValue);
+        try {
+            return gp(pref).getString(key, defaultValue);
+        } catch (ClassCastException e) {
+            return defaultValue;
```

Comment:
Added defensive code to all the get methods in this class. This prevents an exception when `setInt(key)` followed by `getString(key)` etc.